lib/definitions/errors.js

```diff
@@ -1,84 +1,115 @@
-const {inspect} = require('util');
-const {isString} = require('lodash');
-const pkg = require('../../package.json');
+import {inspect} from 'node:util';
+
+import {isString} from 'lodash-es';
+
+const HOMEPAGE = 'https://github.com/semantic-release/github#readme';
```

Comment:
i'm curious about this one. i like the idea of the `package.json` being the source of truth rather than duplicating the value. i know importing json files doesnt work in esm the same way as with cjs, but is it not worth keeping a single source for this?

also, shouldnt the `#readme` still be stripped off the way it was before?

Reply:
> i'm curious about this one. i like the idea of the `package.json` being the source of truth rather than duplicating the value. i know importing json files doesnt work in esm the same way as with cjs, but is it not worth keeping a single source for this?

yeah I had an "ugh" moment here and just decided it's not worth it 🤷🏼 It's not like the homepage is likely to change. And if it is then it would be a new docs website which would many more changes?

Node will get native JSON imports via https://github.com/tc39/proposal-import-assertions

Happy to bring it back the JSON import workaround for now if you feel strong about it

> also, shouldnt the `#readme` still be stripped off the way it was before?

great catch

Reply:
> yeah I had an "ugh" moment here and just decided it's not worth it 🤷🏼

you could convince me that it isnt worth it, but i'm a little more sensitive to it than normal while working to eliminate some of the repetition around version in our docs and dealing with the related broken links. what do you think of this approach?: https://github.com/semantic-release/apm/pull/149/files#diff-0aa43c2bc704def0402bda8d1f61fa08501d3700eff24023d01034e834a4fdb1R6

Reply:
I was able to get this working with `read-pkg` in https://github.com/semantic-release/apm/pull/151/files#diff-0aa43c2bc704def0402bda8d1f61fa08501d3700eff24023d01034e834a4fdb1

Reply:
or if you don't want another dependency you could use `createRequire`

```js
import { createRequire } from "module";
const require = createRequire(import.meta.url);
const pkg = require('../../package.json');
```

Reply:
`read-pkg` won't work here because it's for sync code

It somehow doesn't feel right to bring back `require` just to read out `.homepage` from `package.json`, but I think it's the most pragmatic future safe approach right now, so I'm going to do that